xbmc/utils/URIUtils.h

```diff
@@ -124,14 +124,16 @@ class URIUtils
   static bool IsDOSPath(const std::string &path);
   static bool IsDVD(const std::string& strFile);
   static bool IsFTP(const std::string& strFile);
-  static bool IsHTTP(const std::string& strFile);
+  static bool IsHTTP(const std::string& strFile, bTranslate = false);
```

Comment:
Type of the param is missing! And please see my other comment. Use `bool`, not `const bool&`